### DISCUSSION TITLE
Connection lost. Trying to reconnect... shows up behind footer

### DISCUSSION THREAD

**Discussion by kfalcami** [zauberzeug/nicegui#3835] [Q&A] [answered]
### Question

Just what the title says, screenshot below with the footer in front of the message.  Is there something I can do to make it show up in front of the footer?

![image](https://github.com/user-attachments/assets/6abf6785-303c-4e5d-b31a-64cf21dd077d)

**Comment by falkoschindler** [comment, marked as answer]
Hi @kfalcami, thanks for bringing this up!

NiceGUI's connection popup sets its z-index to 1000, but Quasar's footer gets a z-index of 2000. That's why the popup is behind the footer.

As a workaround you can add this line to each page:
```py
ui.add_css('#popup { z-index: 10000; }')
```

But we should fix it in the library. I'll create an issue.

**Reply by falkoschindler** [reply]
See #3836. It just got released in version 2.4.0 ✔️
